packages/core/lib/commands/packages.js

```diff
@@ -0,0 +1,30 @@
+const { callbackify } = require("util");
+
+var command = {
+  command: "packages",
+  description: "List available packages on connected EthPM Registry",
+  builder: {},
+  help: {
+    usage: "truffle packages",
+    options: []
+  },
+  run: function (options, done) {
+    var Config = require("@truffle/config");
```

Comment:
Another small thing: could you rename these `var`s `const`? Anywhere there is a variable that we won't be setting again it should be `const`. `var`s are kinda old school